cli/src/commands/admin.js

```diff
@@ -0,0 +1,18 @@
+const exec = require('child_process').exec;
+
+module.exports = ui => () =>
+    new Promise((resolve, reject) => {
+        exec('cd ../admin && make start', (error) => {
```

Comment:
Not sure about this, the cli will be installed through npm probably (`npm install comfy-cli` or something like this). It will probably not bundle the other projects.

We need to have a build process which will output the admin compiled for production inside the cli